saige_assoc.py

```diff
@@ -341,12 +347,34 @@ def manage_concurrency_for_job(job: hb.batch.job.Job):
             job.depends_on(jobs[-get_config()['saige']['max_parallel_jobs']])
         jobs.append(job)
 
+    # define writeout file by type of pipeline and date and time
+    date_and_time = datetime.today().strftime('%Y-%m-%d_%H:%M:%S')
+    writeout_file = (
+        f'{writeout_file_prefix}/saige_qtl_common_variant_pipeline_{date_and_time}.csv'
+    )
+
     # pull principal args from config
     chromosomes: list[str] = get_config()['saige']['chromosomes']
     celltypes: list[str] = get_config()['saige']['celltypes']
+    drop_genes: list[str] = get_config()['saige']['drop_genes']
     celltype_jobs: dict[str, list] = dict()
 
     vre_plink_path = f'{vre_files_prefix}/vre_plink_2000_variants'
+    cis_window_size = get_config()['saige']['cis_window_size']
+
+    # populate all the important params into a file for long-term reference
+    writeout_dict: dict = {
+        'ar_guid': try_get_ar_guid() or 'UNKNOWN',
+        'results_output_path': output_path(''),
+        'vre_plink_files_prefix_used': vre_plink_path,
+        'pheno_cov_files_path_used': pheno_cov_files_path,
+        'cis_window_files_path_used': cis_window_files_path,
+        'saige_fit_null_params': get_config()['saige'],
+        'saige_single_variant_test_params': get_config()['saige.sv_test'],
+        'saige_fit_null_job_specs': get_config()['saige.job_specs.fit_null'],
+        'saige_single_variant_test_job_specs': get_config()['saige.job_specs.sv_test'],
+        'runtime_config': getenv('CPG_CONFIG_PATH'),
```

Comment:
FYI the runtime config is the combination of all the default analysis-runner content (references, images, etc.), plus any config files you added using `--config` - it's the file that you read when you use `get_config`

Reply:
ah wait does this mean all of the the flags im adding using get_config() will be added twice?

Reply:
Nope! This is just the path to that file, not the contents! In your output file you'll have some select entries, and the path to this file (if for some reason you wanted to check all the contents in that file)

Reply:
ok perfect! And it will be the actual file in the way it was saved when run, not what's committed in the repo at that stage right? And sorry the AR guid thing exactly what does it tell me? Will it know what commit / branch something was run with?